### DISCUSSION TITLE
4/27(목) Bounds 와 Frame 조사해주세요

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#111] [General]
앞으로 정말 많이 사용할 것입니다. 특히 좌표에 관해서는 snpkit을 사용하지 않을 때는 bounds와 fame는 정말 중요하며 크기 조정 및 사진 조정에서도 진짜 많이 사용합니다. 

만약 Bounds와 Frame를 내가 이제 좀 마스터 한 것 같다 하시면 [링크](https://github.com/jjunhaa0211/PerfectContributionsView) 여기 링크로 가시면 제가 짠 라이브러리가 있을텐데 거기서 생성하는 곳 가면 자신이 이해를 잘했는지 못했는지 확실하게 알 수 있을 것 입니다.

좀 내용이 많을 수 있습니다. 😁

**Comment by Eunho0922** [comment]
넵
